Tests/conf.json

```diff
@@ -2,6 +2,11 @@
     "testTimeout": 160,
     "testInterval": 20,
     "tests": [
+        {
+            "integrations": "GreyNoise",
+            "playbookID": "GreyNoise-Test",
+            "fromversion": "5.0.0"
+        },
```

Comment:
let's remove these lines, partner supported integrations are not tested on our nightly run

Reply:
Sure @guykeller , will remove it